src/components/ChatApp/Settings/Settings.react.js

```diff
@@ -597,7 +609,7 @@ class Settings extends Component {
 		}
 		else{
 			this.setState({
-			selectedSetting: cookies.get('loggedIn')? 'Account':'ChatApp Settings'
+			selectedSetting: cookies.get('loggedIn')? 'General':'ChatApp Settings'
```

Comment:
I missed this change. Where was that change part of the issue? Why was that change? Please revert.